File: Code C++/Employee/Employee.cpp
//San Diego - 28/01/2018 - 20:21.
//Definição das funções da classe Employee.
#include "Employee.h"

//Contrutor, inicializa os atributos dos objetos da classe Employee.
Employee::Employee(string Name, string LastName, int Salr){
	setName(Name);
	setLastName(LastName);
	setSalary(Salr);
}

//Guarda o nome do objeto da classe Employee.
void Employee::setName(string Name){
	name = Name;
}

//Retorna o nome do objeto da classe Employee.
string Employee::getName(){
	return name;
}

//Guarda o sobrenome do objeto da classe Employee.
void Employee::setLastName(string LastName){
	lastName = LastName;
}

//Retorna o sobrenome do objeto da classe Employee.
string Employee::getLastName(){
	return lastName;
}

//Guarda o valor do salário de um objeto da classe Employee.
void Employee::setSalary(int Salr){
	if (Salr < 0)
		salary = 0;

	salary =  Salr;
}

//Retorna o valor do salário de um objeto da classe Employee.
int Employee::getSalary(){
	return salary;
}

File: Learning JAVA/Capítulo 03/Employee/Employee.java
//San Diego - 25/05/2020
// Classe de funcionário

public class Employee{

	//Variáveis de instância.
	private String firstName;
	private String lastName;
	private double salary;

	//Construtor da classe.
	public Employee(String firstName, String lastName, double salary){

		this.firstName = firstName;
		this.lastName = lastName;

		if (salary > 0.0)
			this.salary = salary;
	}

	//Getters e Setters das variáveis de instância
	public String getFirstName(){
		return firstName;
	}

	public void setFirstName(String firstName){
		this.firstName = firstName;
	}

	public String getLastName(){
		return lastName;
	}

	public void setLastName(String lastName){
		this.lastName = lastName;
	}

	public double getSalary(){
		return salary;
	}

	public void setSalary(double salary){
		this.salary = salary;
	}


	}